content/applications/sales/crm/optimize/lead_enrichment.rst

```diff
@@ -0,0 +1,112 @@
+   Purchase)` features before deciding to purchase more credits for the database. This includes
+   demo/training databases, educational databases, and one-app-free databases.
+
+Set up lead enrichment
+======================
+
+To set up lead enrichment in the *CRM* application navigate to :menuselection:`CRM --> Configuration
+--> Settings`. Under the :guilabel:`Lead Generation` section, activate :guilabel:`Lead Enrichment`.
+To activate :guilabel:`Lead Enrichment` tick the box next to :guilabel:`Lead Enrichment`, and select
+either :guilabel:`Enrich leads on demand only` or :guilabel:`Enrich all leads automatically`.
+:guilabel:`Save` the changes.
+
+.. image:: lead_enrichment/lead-enrichment-activate.png
+   :align: center
+   :alt: CRM lead generation settings page, with lead enrichment activation highlighted, and enrich
+         leads on demand only chosen.
+
+Enrich leads
+============
+
+Enrichment of leads is based on the domain of the email address of the customer set on the lead.
+There are two different ways that a lead can be enriched: *automatically* and *manually*.
+
+Automatically enrich leads
+--------------------------
+
+During set up, if :guilabel:`Enrich all leads automatically` was selected, then no action needs to
+be taken by the user to enrich the lead. A scheduled action runs automatically, every sixty minutes,
+and enrichment occurs on leads, after a remote database is contacted.
+
+.. tip::
+   To access the cron that runs for the automatic lead enrichment, navigate, in :ref:`developer mode
+   <developer-mode>`, to :menuselection:`Settings app --> Technical menu --> Automation -->
+   Scheduled Actions`. In the :guilabel:`Search...` type in `partner autocomplete`. Click into the
+   result labeled :guilabel:`Partner Autocomplete Sync with remote DB`, and make any necessary
+   adjustments. For the :guilabel:`Execute Every` field enter a value greater than or equal to five
+   minutes.
+
+.. example::
+   The following is an example of a successful autocomplete of lead enrichment data:
+
+   .. image:: lead_enrichment/lead-enrichment-data.png
+      :align: center
+      :alt: Chatter showing lead enrichment data.
+
+Manually enrich leads
+---------------------
+
+If the :guilabel:`Enrich leads on demand only` option was selected when activating :guilabel:`Lead
+Enrichment`, then each lead that a user wishes to enrich, will need to be manually enriched. This
+is achieved by clicking the :guilabel:`Enrich` button in the top menu of the lead.
+
+The same information will be retrieved at the same :abbr:`IAP (In-App Puchase)` credit cost (one per
+enrichment). This method of enrichment is useful when every lead does not need to be enriched or
+cost is an issue.
+
+.. image:: lead_enrichment/manual-enrichment.png
+   :align: center
+   :alt: Manual enrich button feature highlighted on the CRM lead.
+
+.. tip::
+   Manually enrich leads in bulk using the *list* view. First navigate to user's CRM pipline
```

Comment:
```suggestion
   Manually enrich leads in bulk using the *list* view. First navigate to the user's *CRM* pipeline
```
Do you think CRM should be italicized here, since we're talking about the app? I can't decide, since CRM is also a widespread term. It's up to you